### PR TITLE
Add documentation for CPU profiler.

### PR DESCRIPTION
**Pull request by kenzieschmoll**

### CODE REVIEW

File: docs/timeline.md [hunk 1]
```diff
@@ -58,21 +58,35 @@ When this target is missed, you may experience UI jank or dropped frames.
...
 
-<img src="images/timeline_event_details.png" width="800" />
\ No newline at end of file
+### CPU Flame Chart (Preview)
+This tab of the profiler shows CPU samples for the selected frame event (e.g. VSYNC in the example below). This chart
+should be viewed as a top-down stack trace, where the top-most stack frame calls the one below it, and so on and so forth.
+The width of each stack frame represents the amount of time it consumed the CPU. Stack frames that consume a lot of CPU
+time may be a good place to look for possible performance improvements.
+
+The flame chart supports zooming and panning, and it should be interacted with in the same manner as the Frame Flame Chart.
```
Comment (jacob314) [inline docs/timeline.md]: is this sentence needed? I would think all of our charts will look similar so users will expect the same UI interactions. Feel free to push back if this is useful.

File: docs/timeline.md [hunk 2]
```diff
@@ -58,21 +58,35 @@ When this target is missed, you may experience UI jank or dropped frames.
...
+<img src="images/timeline_frame_flame_chart.png" width="800" />
 
 The flame chart supports zooming and panning. Scroll up and down to zoom in and out, respectively. To pan around, you
-can either click and drag the chart or scroll horizontally. You can also click an event to view more details in the
-details section below the chart.
+can either click and drag the chart or scroll horizontally. You can also click an event to view CPU profiling information
+in the section below the chart.
 
-## Event Details
+## CPU Profiler
```
Comment (jacob314) [inline docs/timeline.md]: maybe add the word preview to this heading?

Comment (kenzieschmoll) [reply docs/timeline.md]: done.

File: docs/timeline.md (hunk 1 above)
Comment (kenzieschmoll) [reply docs/timeline.md]: I agree with you - removed.

**Comment by jacob314** [comment]
Fyi  @sfshaza2.

**Comment by jacob314** [comment]
lgtm

**Comment by kenzieschmoll** [comment]
landing this and will investigate the integration test failure separately.

**Comment by sfshaza2** [comment]
@kenzieschmoll, @jacob314:  Sorry I missed this before, but I was OOO. I just looked through the new/modified text. Had the PR not already been merged, I would have removed "and so on and so forth", and other nits. But it's been merged and it's no on the main website, so I won't nitpick for now.  :)

**Comment by kenzieschmoll** [comment]
Shams - feel free to comment on the PR and I can make another cl with your
edits. Thanks!

**Pull request merged**